engine-src/Game/LambdaHack/Server/Commandline.hs

```diff
@@ -86,6 +87,7 @@ serverOptionsP = do
       sclientOptions = ClientOptions
         { sfonts         = []  -- comes only from config file
         , sfontsets      = []  -- comes only from config file
+        , sfullscreen    = sfullscreenSer
```

Comment:
I think you can remove this line if you name it `sfullscreen` instead of `sfullscreenSer` above. I don't think it's one of the rare cases where we need a separate server and client clone of a setting.

Reply:
Cool, I'm not very familiar with newer GHC features, but I do see the triple dots below which I believe brings in the local vars. Working on this now.

Reply:
It's RecordWildCards + NamedFieldPuns. Enabled by default is the whole codebase. Error-prone, unless you are aware of it and you actively manage it. Then it's quite concise, sometime, but not always, making up for the boilerplate that lens eliminates completely (at it's own cost).